llvm/lib/Target/Xtensa/AsmParser/XtensaAsmParser.cpp

```diff
@@ -193,6 +193,10 @@ struct XtensaOperand : public MCParsedAsmOperand {
 
   bool isImm1_16() const { return isImm(1, 16); }
 
+  bool isImm1n_15() const { return (isImm(1, 15) || isImm(-1, -1)); }
```

Comment:
Extra parens. Not sure I understand the what || isImm(-1, -1) is doing

Reply:
Thank you for review. I removed parens and added a comment.